miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -615,6 +615,19 @@ Sample.hot = {
       .filter(function (kit) { return kit.kitType == 'Extraction'; })
       .map(function (kit) { return kit.name; });
   },
+  
+  /**
+   * Checks if the first sample in the sample being created/updated is an RNA sample
+   */
+  isFirstSampleRNA: function () {
+    var firstSample;
+    if (Sample.hot.createOrEdit == 'Create') {
+      firstSample = Sample.hot.newSamplesJSON[0];
+    } else if (Sample.hot.createOrEdit == 'Edit') {
+      firstSample = Sample.hot.samplesJSON[0];
+    }
+    return firstSample.sampleClassAlias.indexOf("RNA") != -1;
```

Comment:
We might want to create a table of SampleClass to QC relations. Not for this request though.